Bottom: 8eb7c4bfcb412c49cebf82c61adb71fcfcbbe7ca
Top:    d7fb817934d15b891eb82bbbd43be431018ad4e6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-23 00:44:35 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
